entries/p/postype.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Postype": {
+    "domain": "postype.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.postype.com/hc/ko/articles/21876052282009-2%EB%8B%A8%EA%B3%84-%EC%9D%B8%EC%A6%9D-%EC%84%A4%EC%A0%95-%ED%95%B4%EC%A0%9C",
+    "categories": [
+      "entertainment",
+      "social"
```

```suggestion
      "retail"
```

From what I could glean from their [about page](https://www.postype.com/about), it seems to be more of a marketplace.

---

In a sense? It’s more like a patreon thing where you can pay creators to access content. There is a physical store but that’s a recent addition and still beta.

---

In that case, let's go with `crowdfunding` and `retail` as the categories.